starter-project-web-vue2/store/luhana.js

```diff
@@ -0,0 +1,33 @@
+import axios from 'axios'
+
+export const state = {
+  blogs: [],
+}
+export const actions = {
+  async fetchBlogs({ commit }) {
+    const response = await axios.get(
+      'https://blog-app-backend.onrender.com/api/articles/all'
```

Comment:
we have already set the `baseURL` in nuxt config. so the relative url is enough. `await this.$axios.get('articles/all')`